packages/terra-slide-panel-manager/src/SlidePanelManager.jsx

```diff
@@ -31,6 +36,7 @@ const propTypes = {
 };
 
 const defaultProps = {
+  level: 1,
```

Comment:
Default should be 2 or 3. Since it's within the same context as the main page, it should be a subheading of the main page, which should only have one h1.

Reply:
Changed to default level 2 [ca3f033](https://github.com/cerner/terra-framework/pull/1621/commits/ca3f0331cd8ced41a275a64b33cc4a1fc91ec6ee)